lib/api/AuthTokensController.cpp

```diff
@@ -64,16 +65,19 @@ namespace MAT_NS_BEGIN {
 
     std::vector<std::string>&  AuthTokensController::GetTickets()
     {
+        LOCKGUARD(m_lock);
```

Comment:
The mutex is kinda pointless - since the reference is returned, outside code could perform concurrent modification of the vector contents without a lock......